controllers/host/platformnetworks.go

```diff
@@ -439,11 +437,34 @@ func (r *HostReconciler) IsReconfiguration(client *gophercloud.ServiceClient, ne
 	return nil, false
 }
 
-// ShouldReconcile is a very important function that really controls the reconciliation
+func (r *HostReconciler) ReconcileStaticPlatformNetworkExpected(client *gophercloud.ServiceClient,
+	network_instance *starlingxv1.PlatformNetwork,
+	addrpool_instance *starlingxv1.AddressPool,
+	uuid string) (error, bool) {
+
+	if addrpool_instance != nil {
+		err, is_reconfig := r.IsReconfiguration(client, network_instance, addrpool_instance)
+		if err != nil {
+			return err, false
+		}
+		if !is_reconfig {
+			return nil, true
+		}
+	} else {
+		// for networks
```

Comment:
What does this comment mean?

Reply:
Updated the comment as follows:
> // Since addrpool_instance is nil the ReconcileOtherPlatformNetworksExpected
> 		// call is implied for networks.
> 		// If UUID is empty, it means network has to be created, hence return true.